zerver/views/__init__.py

```diff
@@ -572,6 +572,36 @@ def dev_direct_login(request, **kwargs):
     return HttpResponseRedirect("%s%s" % (settings.EXTERNAL_URI_SCHEME,
                                           request.get_host()))
 
+@csrf_exempt
+@require_post
+@has_request_variables
+def api_dev_android_direct_login(request, username=REQ):
```

Comment:
It would work If I replace the REQ with REQ() as I see these changes have been implemented in this commit https://github.com/zulip/zulip/commit/960144a49e66bd330daecdf3423e3a53c8353352 , 
So I should just replace those REQ right?